server.py

```diff
@@ -88,5 +90,6 @@ def healthcheck():
     logging.basicConfig(level=settings.LOGGING_LEVEL, format=settings.LOGGING_FORMAT)
     handler = logging.StreamHandler(sys.stdout)
     app.logger.addHandler(handler)
+    app.logger.info("Starting server: version='{}'".format(__version__))
```

Comment:
The way this was originally written didn't use `structlog` to wrap the `logger` object. Rather than introduce new libraries, I've written a string that will emulate the output of the other services. If future requirements introduce more log key:value pairs, we can review this and potentially introduce `structlog` then.

Printed output will be:

```
2017-03-15 11:34:25,982|INFO: sdx-sequence: Starting server: version='1.3.0'
```

I'm not sure why a new `StreamHandler` is added to the `logger` - this is already instantiated by the call to `basicConfig`. In future releases it is worth exploring whether this can be refactored - it is essentially duplicating logs to stdout.